### PR TITLE
Rename AWS Bedrock integration as Amazon Bedrock

### PR DESCRIPTION
**Pull request by agithomas**
Bug

## Proposed commit message

- Rename AWS Bedrock as Amazon Bedrock.
- Change the minimum supported stack version to `8.13` in the document.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [x] Modified dashboard
- [x] Modified dashboard screenshot
- [x] Modified Manifest file
- [x] Modified Ingest pipeline descriptions

## How to test this PR locally

- `elastic-package build`
- `elastic-package stack up -v -d --services package-registry`
- Use Amazon Bedrock playground for integration testing

## Related issues
- https://github.com/elastic/integrations/issues/11180 

## Screenshots
Screenshots are part of the change

### CODE REVIEW

**Review by ShourieG — APPROVED** [review]
LGTM

**Comment by ShourieG** [comment]
@agithomas, we might need to regenerate the tests as it will contain the input name as aws bedrock

File: packages/aws_bedrock/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "0.8.0"
+  changes:
+    - description: Updated name of the integration as Amazon Bedrock.
```
Comment (muthu-mps) [inline packages/aws_bedrock/changelog.yml:3]:
```suggestion
    - description: Update integration name to Amazon Bedrock.
```

File: packages/aws_bedrock/changelog.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
+- version: "0.8.0"
```
Comment (muthu-mps) [inline packages/aws_bedrock/changelog.yml:1]: Can we consider patch release and update version to 0.7.1?

File: packages/aws_bedrock/changelog.yml (hunk 1 above)
Comment (agithomas) [reply packages/aws_bedrock/changelog.yml]: Updated

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**